src/Installer/redist-installer/projects/VSTemplateLocator/VSTemplateLocator.csproj

```diff
@@ -21,10 +21,13 @@
     <ProjectReference Include="$(RepoRoot)src\Microsoft.DotNet.TemplateLocator\Microsoft.DotNet.TemplateLocator.csproj" ReferenceOutputAssembly="false" />
   </ItemGroup>
 
-  <Target Name="GenerateLayout"
-          Condition="'$(OS)' == 'Windows_NT' and '$(_SuppressAllTargets)' != 'true'"
```

Comment:
Any idea why this was here? Old build optimization?

Reply:
You mean the `OS` check? Yes I guess so because the layout is only ever used in win-x64.